modules/custom/cu_help/cu_help.module

```diff
@@ -1,298 +1,44 @@
-    'title' => 'Contact Information Help',
-    'short_title' => 'Contact Information',
-    'url' => '/node/408',
-  );
-
-  $variables['admin/settings/site-configuration/error'][] = array(
-    'title' => 'Error Pages Help',
-    'short_title' => 'Error Pages',
-    'url' => '/node/402',
-  );
-
-  $variables['admin/config/search/redirect'][] = array(
-    'title' => 'Adding Redirects Help',
-    'short_title' => 'Adding Redirects',
-    'url' => '/node/584',
-  );
-
-  $variables['admin/settings/site-configuration/site-name'][] = array(
-    'title' => 'Site Name Help',
-    'short_title' => 'Site Name',
-    'url' => '/node/408',
-  );
-
-  $variables['admin/settings/site-configuration/google-analytics'][] = array(
-    'title' => 'Google Analytics Help',
-    'short_title' => 'Google Analytics',
-    'url' => '/node/400',
-  );
-
-  $variables['admin/settings/bundles/list'][] = array(
-    'title' => 'Adding Bundles',
-    'short_title' => 'Adding Bundles',
-    'url' => '/node/656',
-  );
-
-  $variables['admin/settings/bundles/list/addon'][] = array(
-    'title' => 'Contact Web Support',
-    'short_title' => 'Contact Web Support',
-    'url' => '/node/78',
-  );
-
-  $variables['admin/settings/bundles/list/request'][] = array(
-    'title' => 'Contact Web Support',
-    'short_title' => 'Contact Web Support',
-    'url' => '/node/78',
-  );
-
-  $variables['admin/settings/cache/clear'][] = array(
-    'title' => 'Understanding Cache Clearing',
-    'short_title' => 'Cache Clearing',
-    'url' => '/node/1350',
-  );
-
-  $variables['admin/settings/cache/clear/varnish-path'][] = array(
-    'title' => 'Understanding Cache Clearing',
-    'short_title' => 'Cache Clearing',
-    'url' => '/node/1350',
-  );
-
-  $variables['admin/settings/cache/clear/drupal-full'][] = array(
-    'title' => 'Understanding Cache Clearing',
-    'short_title' => 'Cache Clearing',
-    'url' => '/node/1350',
-  );
-
-  $variables['admin/settings/cache/clear/varnish-full'][] = array(
-    'title' => 'Understanding Cache Clearing',
-    'short_title' => 'Cache Clearing',
-    'url' => '/node/1350',
-  );
-
-  $variables['admin/settings/search/search-settings'][] = array(
-    'title' => 'Search Box Help',
-    'short_title' => 'Search Box',
-    'url' => '/node/1670',
-  );
-
-  $variables['admin/people/invite'][] = array(
-    'title' => 'Invite Users Help',
-    'short_title' => 'Invite Users',
-    'url' => '/node/654',
-  );
-
-  $variables['admin/people/invite/operations'][] = array(
-    'title' => 'Manage Invites Help',
-    'short_title' => 'Manage Users',
-    'url' => '/node/1946',
-  );
-
-  return array_map(
-    function ($a) {
-      $a[0]['module'] = 'cu_help';
-      return $a;
-    },
-    $variables);
+function cu_help_menu_alter(&$items) {
+  unset($items['admin/help']);
 }
 
 /**
- * Implements hook_express_help_fields().
+ * Implements hook_navbar().
+ *
+ * Add Help link to navbar.
```

Comment:
Please add a comment that indicates that this is intentionally duplicated code and reference the ticket for removing the older code in the next release.